examples/templates/aws-linux/main.tf

```diff
@@ -36,6 +36,22 @@ variable "region" {
   }
 }
 
+variable "instance_type" {
+  description = "What instance type should your workspace use?"
+  default     = "2C/1G: t3.micro"
+  validation {
+    condition = contains([
+      "2C/1G: t3.micro",
+      "2C/2G: t3.small",
```

Comment:
It wasn't immediately clear what "C" and "G" meant in this context.  I eventually realized it was vCPU and GB RAM, but I think 2 vCPU /  2 GB RAM is clearer and it's not like we're hurting for space in this context, nor do the users have to type these values in our UX.

Unfortunately the vCPU number is a little misleading for T3 instances. Micro-large have the same number of vCPUs, and reading this you might conclude that they have the same CPU performance.  They do not.  Amazon throttles the CPUs except for short bursts and the amount of throttling is different for the different instance sizes.  I'm not really sure what to suggest except to drop the stats and just include the instance type names.  Up to you.

Reply:
I can drop the stats, I only put them in there so the instance types would show up in the right order :D

Reply:
I guess you could do numbered list, `1. t3.micro`, `2. t3.small` if you want to maintain order? 😄